recipes/lp_solve/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set name = "lp_solve" %}
+{% set version = "5.5.2.5" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}    
+
+source:
+    fn: {{ name|lower }}_{{ version }}_source.tar.gz
+    url: https://sourceforge.net/projects/lpsolve/files/lpsolve/{{ version }}/{{ name|lower }}_{{ version }}_source.tar.gz
+    md5: 3be57261fc41dd8e210f54017220d5f7
+
+build:
+    number: 0
+
+test:
+    files:
+      - demo.c
+
+about:
+    home: http://lpsolve.sourceforge.net/5.5/
+    summary: lp_solve is a Mixed Integer Linear Programming (MILP) solver
+    license: LGPL
```

Comment:
Then you have to put a copy in the recipe. See http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually